gr-filter/include/gnuradio/filter/mmse_interp_differentiator_ff.h

```diff
@@ -37,7 +37,6 @@ class FILTER_API mmse_interp_differentiator_ff
 {
 public:
     mmse_interp_differentiator_ff();
-    ~mmse_interp_differentiator_ff();
 
```

Comment:
Missed a `const` below.

Reply:
Indeed.

Any guesses on why the move constructors have to be explicitly defaulted? [Reads to me](https://en.cppreference.com/w/cpp/language/move_constructor) under these circumstances.

I'll add your changes anyway.